com.Adobe.Flash-Player-Projector.json

```diff
@@ -0,0 +1,64 @@
+{
+    "app-id" : "com.Adobe.Flash-Player-Projector",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "1.6",
+    "sdk" : "org.freedesktop.Sdk",
+    "command" : "flashplayer",
+    "finish-args" : [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Play sounds */
+        "--socket=pulseaudio"
+    ],
+    "modules" : [
+        {
+            "name" : "flashplayer",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "mkdir /app/bin",
+                "install apply_extra /app/bin",
+                "install flashplayer /app/bin",
+                "install -m0644 -D com.Adobe.Flash-Player-Projector.svg /app/share/icons/com.Adobe.Flash-Player-Projector.svg",
+                "install -m0644 -D com.Adobe.Flash-Player-Projector.desktop /app/share/applications/com.Adobe.Flash-Player-Projector.desktop",
+                "install -m0644 -D com.Adobe.Flash-Player-Projector.appdata.xml /app/share/metainfo/com.Adobe.Flash-Player-Projector.appdata.xml"
+            ],
+            "sources" : [
+                {
+                    "type" : "file",
+                    "path" : "flashplayer"
```

Comment:
You can merge this into the manifest by using source type "script".

Reply:
And what would the escaping look like?

Reply:
Done, but see below.